custom-payment-flow/client/html/link.js

```diff
@@ -0,0 +1,80 @@
+document.addEventListener('DOMContentLoaded', async () => {
+  // Load the publishable key from the server. The publishable key
+  // is set in your .env file.
+  const {publishableKey} = await fetch('/config').then((r) => r.json());
+  if (!publishableKey) {
+    addMessage(
+      'No publishable key returned from the server. Please check `.env` and try again'
+    );
+    alert('Please set your Stripe publishable API key in the .env file');
+  }
+
+  const stripe = Stripe(publishableKey, {
+    apiVersion: '2020-08-27',
```

Comment:
wonder if we should update all of these to the latest `2022-11-15` at some point.

Reply:
I'm updating the link file for now and can open another issue to do it in other files